### ISSUE TITLE
RequireJavadoc false positive

### ISSUE THREAD

**Issue by wmdietl** [eisop/checker-framework#1814]
PR #1813 touches file framework/src/main/java/org/checkerframework/framework/util/typeinference8/types/Qualifier.java and misc fails with:

```
+ /home/runner/work/checker-framework/checker-framework/checker/bin-devel/.plume-scripts/lint-diff.py --guess-strip /tmp/diff-IdY6Sh.txt /tmp/warnings-requireJavadoc.txt
/home/runner/work/checker-framework/checker-framework/framework/src/test/java/org/checkerframework/framework/testchecker/supportedquals/qual/Qualifier.java:9:1: missing documentation for Qualifier
+ status=1
```

It looks like only the basename `Qualifier.java` is used when looking for missing javadoc, leading to this false positive.
